apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -253,6 +299,17 @@ import { dialectFor, FormatterDialect } from '@shared/lib/dialects/models'
 import { format } from 'sql-formatter';
 import { normalizeFilters, safeSqlFormat } from '@/common/utils'
 import { TableFilter } from '@/lib/db/models';
+import Noty from 'noty'
```

Comment:
I think it'd be good to break out this code into a separate component, maybe call it `JsonEditorModal.vue` or something literal like that.

Reply:
Agreed